connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/UserDefinedFunctionE2ETestSuite.scala

```diff
@@ -346,4 +347,42 @@ class UserDefinedFunctionE2ETestSuite extends QueryTest {
     val result = df.select(f($"id")).as[Long].head()
     assert(result == 1L)
   }
+
+  test("UDAF custom Aggregator - primitive types") {
```

Comment:
Can you add a test for a UDAF with a custom toColumn implementation?

Reply:
Let's do this in the next PR where we add support of `toColumn`.